providers/tests/snowflake/operators/test_snowflake_sql.py

```diff
@@ -27,15 +28,19 @@
 
 databricks = importorskip("databricks")
 
+MockRow = None
 try:
     from databricks.sql.types import Row
 except ImportError:
```

Comment:
The goal is to remove this import. We should not import from Databricks in the Snowflake tests

Reply:
sorry for the late update, I pushed changes, but I'm having some trouble running the updated test on my local environment